src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleaner.py

```diff
@@ -765,3 +808,34 @@ def alertDeletablePU(self, workflowName, containerName, ruleList):
         self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription,
                        service=self.alertServiceName, endSecs=self.alertExpiration)
         self.logger.critical(alertDescription)
+
+    def alertStatusAdvanceExpired(self, wflow, additionalInfo=None):
+        """
+        Send alert notifying that a workflow has spend too much time in a given status
+        :param wflow: MSRuleCleanerWorkflow instance
+        :additionalInfo: String with additional information.
+        :return: none
+        """
+        # Check if alarms are enabled for this service:
+        if not self.msConfig["sendNotification"]:
```

Comment:
Todor, please move this check to the very end of this alert, such that we still get the log record in the service logs. Thus, something like:
```
...
        self.logger.critical(alertDescription)
        if not self.msConfig["sendNotification"]:
            # alert to expiry in 2 days
            self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription,
                           service=self.alertServiceName, endSecs=self.alertExpiration)
```